webpack.common.js

```diff
@@ -44,9 +44,6 @@ module.exports = {
 	},
 	plugins: [new VueLoaderPlugin(), new StyleLintPlugin()],
 	resolve: {
-		alias: {
```

Comment:
@skjnldsv This might also be an option for the contacts app. See https://blog.wuc.me/2018/10/18/nextcloud-vue-csp.html and https://vuejs.org/v2/guide/installation.html#Runtime-Compiler-vs-Runtime-only. For Tasks, this reduced the dev build by ~200 kB.

Reply:
@raimund-schluessler Didn't we already do that? :O :D